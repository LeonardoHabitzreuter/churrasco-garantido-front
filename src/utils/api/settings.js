export const BASE_API_URL = process.env.apiHost || 'http://localhost:3001'
