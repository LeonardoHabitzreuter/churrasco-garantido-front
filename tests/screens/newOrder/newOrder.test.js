import React from 'react'
import { shallow } from 'enzyme'
import NewOrder from '../../../src/screens/newOrder'
import api from '../../../src/utils/api'
api.getUser = jest.fn()

describe('New order component', () => {
  test('should renders properly', () => {
    const wrapper = shallow(<NewOrder />)
    expect(wrapper).toMatchSnapshot()
  })
})
