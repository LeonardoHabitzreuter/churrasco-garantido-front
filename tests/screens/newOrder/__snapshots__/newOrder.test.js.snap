// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New order component should renders properly 1`] = `
<div
  className="col-sm-8 full-width"
  style={
    Object {
      "display": "inline-grid",
    }
  }
>
  <PageHeader
    bsClass="page-header"
  >
    Novo pedido
  </PageHeader>
  <Loader
    loading={true}
    size="large"
  />
  <Alert
    handleDismiss={[Function]}
    messages={Array []}
    show={false}
    style=""
  />
  <div>
    <InputGroup
      labelName="Empresa"
    >
      <Dropdown
        id="companiesDropdown"
        items={Array []}
        onChange={[Function]}
        selectedIndex={null}
      />
    </InputGroup>
    <InputGroup
      labelName="Produto"
    >
      <Dropdown
        id="productsDropdown"
        items={
          Array [
            "Cerveja",
            "Carne",
            "Pão de alho",
            "Refrigerante",
            "Guardanapo",
            "Carvão",
            "Alcool",
            "Churrasqueira",
          ]
        }
        onChange={[Function]}
        selectedIndex={0}
      />
    </InputGroup>
    <InputGroup
      labelName="Quantidade"
    >
      <input
        min="1"
        onChange={[Function]}
        type="number"
        value={1}
      />
    </InputGroup>
    <InputGroup
      labelName="1 Cerveja"
    >
      <Button
        bsStyle="success"
        onClick={[Function]}
        type="button"
      >
        Adicionar
      </Button>
    </InputGroup>
  </div>
  <div
    style={
      Object {
        "paddingTop": "5rem",
      }
    }
  >
    <Table
      columns={
        Object {
          "amount": "Quantidade",
          "product": "Produto",
        }
      }
      lines={Array []}
    />
  </div>
  <Button
    bsStyle="primary"
    className="col-sm-2"
    onClick={[Function]}
    type="button"
  >
    Fechar pedido
  </Button>
</div>
`;
